src/app/workspace/resource/resource.component.html

```diff
@@ -34,7 +34,7 @@
         </div>
 
         <!-- tabs -->
-        <mat-tab-group animationDuration="0ms" [(selectedIndex)]="selectedTab" (selectedTabChange)="tabChanged($event)">
+        <mat-tab-group *ngIf="!resource.res.isDeleted; else deletedResource" animationDuration="0ms" [(selectedIndex)]="selectedTab" (selectedTabChange)="tabChanged($event)">
```

Comment:
I think we should also have a similar `ngIf` for the representation div of this file. As it is now, if I delete a resource with some sort of resource representation (audio, document, image, archive, etc.), I still see the representation. The resource representation is only hidden once I refresh the page.

Reply:
oh yes, you're right. I'll update asap

Reply:
Yes, that's right. When deleting a resource, it's still visible as it was before (incl. all property values etc) because I do not get the updated resource information yet. In comparison to the current view of a deleted resource does not have any property and value information. But Rebecca is working on a better version, that we still get the whole resource information even when it's deleted. The current implementation in this PR is only a temporary solution to display something. In the previous version, the user was getting a blank page only.